### PR TITLE
Update dependencies (spring boot vulnerability)

### PR DESCRIPTION
**Pull request by steverendell**
Also remove some unused dependencies

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -80,9 +78,9 @@ dependencies {
...
 
-    runtime 'com.h2database:h2:1.4.200'
-    runtime 'org.flywaydb:flyway-core:6.1.0'
-    runtime 'org.postgresql:postgresql:42.2.9'
+    runtimeOnly 'com.h2database:h2:1.4.200'
+    runtimeOnly 'org.flywaydb:flyway-core:6.2.0'
+    runtimeOnly 'org.postgresql:postgresql:42.2.9'
 
     implementation("org.springframework.boot:spring-boot-starter-web")
     // implementation("org.springframework.boot:spring-boot-devtools")
```
Comment (brightonsbox) [inline build.gradle]: Remove this?

File: build.gradle [hunk 2]
```diff
@@ -92,37 +90,36 @@ dependencies {
...
-    implementation 'net.logstash.logback:logstash-logback-encoder:6.2'
+    implementation 'net.logstash.logback:logstash-logback-encoder:6.3'
     implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:2.6.0-BETA'
     implementation 'com.microsoft.azure:applicationinsights-logging-logback:2.6.0-BETA'
     agentDeps 'com.microsoft.azure:applicationinsights-agent:2.6.0-BETA'
 
     implementation 'javax.annotation:javax.annotation-api:1.3.2'
-    implementation 'javax.xml.bind:jaxb-api:2.3.0'
-    implementation 'com.sun.xml.bind:jaxb-impl:2.3.2'
+    implementation 'javax.xml.bind:jaxb-api:2.4.0-b180830.0359'
```
Comment (brightonsbox) [inline build.gradle]: Will these -b prefixes mean they are beta versions?

File: build.gradle (hunk 1 above)
Comment (steverendell) [reply build.gradle]: ah ok, will do next time

File: build.gradle (hunk 2 above)
Comment (steverendell) [reply build.gradle]: Ive just gone with :dependencyUpdates's recommendations

Comment (petergphillips) [reply build.gradle]: agreed, doesn't seem right to me.  Looks like this project doesn't have the latest dependency update configuration so is suggesting beta versions too.  Worth therefore copying the configuration from a different project - e.g. nomis-oauth2-server.  Even so it sometimes suggests beta versions which we then tend to ignore ..

Comment (steverendell) [reply build.gradle]: Done.

**Pull request merged**